### DISCUSSION TITLE
Questions about connecting multiple time-series databases to Nightingale

### DISCUSSION THREAD

**Discussion by langsile0** [ccfos/nightingale#1257] [General]
还没使用夜莺，在看了文档之后有个疑问，这里接入多个prometheus，是把prometheus当作数据存储来使用了吗，采集器收集数据写入prometheus。
我有两个prometheus，数据不同，这种情况可以通过夜莺来展示数据，配置告警吗

**Comment by UlricQin** [comment]
已有的prometheus可以直接接入：https://mp.weixin.qq.com/s/ThFjha8ocjdz3sqI1Pjwjg
